target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -679,6 +679,12 @@ tl-wr941nd-v6)
 	ucidef_set_led_switch "lan4" "LAN4" "tp-link:blue:lan4" "switch0" "0x02"
 	ucidef_set_led_wlan "wlan" "WLAN" "tp-link:blue:wlan" "phy0tpt"
 	;;
+tl-wr840n-v2|\
+tl-wr840n-v3)
+	ucidef_set_led_netdev "wan" "WAN" "tp-link:green:wan" "eth1"
+	ucidef_set_led_netdev "lan" "LAN" "tp-link:green:lan" "eth0"
```

Comment:
Use eth0 as lan led trigger is bad idea, because eth0 interface is on when br-lan is up. Better to use:
ucidef_set_led_switch "lan" "LAN" "tp-link:green:lan" "switch0" "0x1E"